client/src/components/ApplicationDashBoard/ApplicationDashBoard.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { useAppSelector } from "../../app/hooks";
+
+interface IStatusCount {
+  status: string;
+  count: number;
+}
+
+const ApplicationDashboard = (): JSX.Element => {
+  const [totalApplications, setTotalApplications] = useState(0);
+  const [applicationsByStatus, setApplicationsByStatus] = useState<
+    IStatusCount[]
+  >([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+  const user = useAppSelector((state) => state.user.userData);
+
+  useEffect(() => {
+    async function fetchAggregatedData() {
+      setLoading(true);
+      try {
+        const response = await axios.get(
+          `/api/applications/aggregated-user-stats/${user?._id}`
+        );
+        const { totalApplications = 0, applicationsByStatus = [] } =
+          response.data || {};
+        setTotalApplications(totalApplications);
+        setApplicationsByStatus(applicationsByStatus);
+        setLoading(false);
+      } catch (err) {
+        const error = err as Error;
+        console.error("Error fetching aggregated data:", error);
+        setError(error.message);
+        setLoading(false);
+      }
+    }
+
+    fetchAggregatedData();
```

Comment:
Because this is just for a user to get their own stats, and stats can only be updated by the user - I'm thinking there's a way to store this state with redux and check if we've already fetched it - save us from sending this request everytime this component renders

Reply:
100% The state for the Dashboard will be moving to Redux. The only thing I would ideally want to be handled by local states for the most part are temp input fields etc..